java/pom.xml

```diff
@@ -482,6 +482,38 @@
             </lifecycleMappingMetadata>
           </configuration>
         </plugin>
+
+        <plugin>
+          <groupId>org.xolstice.maven.plugins</groupId>
+          <artifactId>protobuf-maven-plugin</artifactId>
+          <version>0.6.1</version>
+          <configuration>
+            <protocArtifact>com.google.protobuf:protoc:${dep.protobuf.version}:exe:${os.detected.classifier}</protocArtifact>
+            <pluginId>grpc-java</pluginId>
+            <pluginArtifact>io.grpc:protoc-gen-grpc-java:${dep.grpc.version}:exe:${os.detected.classifier}</pluginArtifact>
+          </configuration>
+          <executions>
```

Comment:
Executions are usually not specified in `pluginManagement` section, only common configuration which should apply to all executions. For example, `${basedir}` is relative to the current project, so `${basedir}/../format` would be invalid in most modules.

Reply:
You are right, fixed!